ramalama/common.py

```diff
@@ -17,12 +17,6 @@ def in_container():
 
 
 def container_manager():
```

Comment:
In some ways it's a pity this will attempt to use podman on macOS by default...

macOS native will probably outperform Linux guest VM from using native APIs (and just macOS being very mature here from lots of usage, llama.cpp for example was first created to run on macOS and Apple Silicon) and will have access to more memory... And it's makes ramalama more complex to start on macOS... One has to start/create a podman-machine with enough memory, disk, etc. Could potentially kill the "boring" aspect on macOS.

I do think we should leave this in until we are at least sure we work with ramalama via podman-machine on macOS out of the box accelerate, we aren't far away from that, but I don't think we've tested that end to end.

I did get good feedback from everyone that used macOS native ramalama so far.

Strong chance this is why the build is failing on macOS but didn't check thoroughly.

Reply:
`ramalama currently only works on linux and macOS`

should be the below string...